common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -881,4 +886,25 @@ private static void validateExtIdTypeAndProvider(Request userRequest) {
           ERROR_CODE);
     }
   }
+
+  private static void checkForDuplicateExternalId(List<Map<String, String>> list) {
+    List<Map<String, String>> tempList = new ArrayList<>();
+    if (CollectionUtils.isNotEmpty(list)) {
```

Comment:
Remove this as it will always cause duplicate error if multiple external IDs are provided.